header-validator/validate-json.js

```diff
@@ -176,22 +213,24 @@ const aggregationKeys = object((state, key, value) => {
 export function validateSource(source) {
   const state = new State()
   state.validate(source, {
-    aggregation_keys: optional(aggregationKeys),
-    debug_key: optional(uint64),
-    destination: required(destination),
-    filter_data: optional(filters(/*allowSourceType=*/ false)),
-    priority: optional(int64),
-    source_event_id: required(uint64),
+    [sourceFields.AGGREGATION_KEYS]: optional(aggregationKeys),
+    [sourceFields.DEBUG_KEY]: optional(uint64),
+    [sourceFields.DESTINATION]: required(destination),
+    [sourceFields.FILTER_DATA]: optional(filters(/*allowSourceType=*/ false)),
+    [sourceFields.PRIORITY]: optional(int64),
+    [sourceFields.SOURCE_EVENT_ID]: required(uint64),
```

Comment:
Instead of referencing the `sourceFields` object keys like this, can we instead lift up this entire dictionary so it's:

```js
const sourceFields = Object.freeze({
  'aggregation_keys': optional(aggregationKeys),
  'debug_key': optional(uint64),
  ...
})
```

Reply:
Using regular functions sounds good to me.